blocks/network-item/network-item.css

```diff
@@ -0,0 +1,87 @@
+.network-item div.tags {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 24px;
```

Comment:
Nit: px to rem

Reply:
Fair point, should I update the sizes in the `hero-horizontal-tabs` as well or leave those as is for the moment?